crates/context_aware_config/src/api/default_config/types.rs

```diff
@@ -10,6 +12,38 @@ pub struct CreateReq {
     pub function_name: Option<Value>,
 }
 
+#[derive(Debug, Deserialize, AsRef, Deref, DerefMut, Into)]
+#[serde(try_from = "String")]
+pub struct DefaultConfigKey(String);
```

Comment:
shouldn't this be simply `ConfigKey`, seems a bit confusing naming it specifically `DefaultConfigKey`, or do we have some specific reason to name it this way ?

Reply:
Generally for all types I followed entityname<>fieldname